### PR TITLE
Creating view all articles functionality

### PR DESCRIPTION
**Pull request by addoadil**
I do want to mention that on my end when I first start my localhost server to see the changes on screen it does take 1 to 2 mins for all assets to render on my page. Once the first render is done then every time I click refresh everything is then rendered instantaneously. I do intend to add loading later on this is just FYI.

### CODE REVIEW

**Review by robert112233 — CHANGES_REQUESTED** [review]
Great work Adil! I like the stying and layout of the list. You have a warning in the browser console which you should fix. 

Also you could think about adding more information on each article, properties like the date, comment count and author will be very useful for when you sort the articles, otherwise how would a user know it's worked? 
Things like comment count also give a better indication on which articles are popular, providing a better UX.

Please make the changes and submit another PR for review 👍

File: components/PopularArticlesList.jsx [hunk 1]
```diff
@@ -20,16 +20,33 @@ function PopularArticlesList() {
     <div className="container">
       <div className="article-list">
         {articles.map((article) => {
-          const { article_id, title, article_img_url } = article;
+          console.log(article);
```
Comment (MKD01) [inline components/PopularArticlesList.jsx]: Dont forget to remove console.logs

File: components/PopularArticlesList.jsx [hunk 2]
```diff
@@ -20,16 +20,33 @@ function PopularArticlesList() {
...
+          const {
+            article_id,
+            title,
+            article_img_url,
+            author,
+            comment_count,
+            created_at,
+          } = article;
+          const publishedDate = created_at.substring(0, 10);
+          console.log(publishedDate);
```
Comment (MKD01) [inline components/PopularArticlesList.jsx]: This one too

**Review by MKD01 — APPROVED** [review]
Great work Just a couple console.logs to remove and your good to merge this in 👍

**Pull request merged**